ex1/functions.py

```diff
@@ -65,24 +78,48 @@ def f():
 # compare against. The function should return a list with equal length as the
 # input list containing for each number in the original list either True or
 # False if the number was greater than or equal to n.
-
+def check_list(numlist, num):
+    for i in range (len(numlist)):
+        if numlist[i] >= num:
+            numlist[i] = True
+        else:
+            numlist[i] = False
+    return numlist
 # Define a function named check_list_nth that does the same as check_list but
 # uses every nth element of the input list (including the first one). You will
 # need a third input argument.
-
+def check_list_nth(numlist, num, nth):
+    newlist = []
+    for i in range (0, len(numlist), nth):
+        if numlist[i] >= num:
+            newlist.append(True)
```

Comment:
You can do this in a list comprehension as e.g. [here](https://github.com/TUW-GEO-python-intro/Exercise-1/pull/20/files#diff-7ec7b04e6cd2fbdbc70b7b64bf731ae8R188). Or you can directly append `numlist[i] >= num` instead of the `if else` block.